config/locales/diaspora/en.yml

```diff
@@ -921,68 +921,68 @@ en:
         picture:
-          name: "picture"
-          description: "This grants read access to user's profile picture data to the application."
+          name: "Profile picture"
+          description: "This grants read-only access to your profile picture."
         'contacts:read':
-          name: "contacts:read"
-          description: "This grants read permissions to contacts and related data (like aspects) to the application."
+          name: "Contacts (Read-only)"
+          description: "This grants read-only access to contacts and related data (like aspects)."
         'contacts:modify':
-          name: "contacts:modify"
-          description: "This grants write permissions to contacts and related data (like aspects) to the application."
+          name: "Contacts (Write)"
+          description: "This grants write access to contacts and related data (like aspects)."
         conversations:
-          name: "conversations"
-          description: "This grants read/write permission to private messaging to the application."
+          name: "Conversations"
+          description: "This grants read and write permission to private messages."
         email:
-          name: "email"
-          description: "This grants read access to the user's email address to the application."
+          name: "E-Mail"
+          description: "This grants read-only access to your email address."
         interactions:
-          name: "interactions"
-          description: "This grants read and write permissions on interacting with posts to the application."
+          name: "Interactions"
+          description: "This grants access to interact with posts, like liking or submitting a comment."
         notifications:
-          name: "notifications"
-          description: "This grants read and write permissions on notifications for the user to the application."
+          name: "Notifications"
+          description: "This grants read and write access to your notifications."
         'private:read':
-          name: "private:read"
-          description: "This grants read privileges on private posts and streams to the application."
+          name: "Private Posts (Read-only)"
+          description: "This grants read-only access to your and your contacts' private posts."
         'private:modify':
-          name: "private:modify"
-          description: "This grants modification privileges on private posts and streams, including allowing private sharing, to the application."
+          name: "Private Posts (Write)"
+          description: "This grants access to publish private posts."
         'public:read':
-          name: "public:read"
-          description: "This grants read permissions on public posts, interactions, and user data to the application."
+          name: "Public Posts (Write)"
+          description: "This grants access to your public posts, and their interactions and related data."
```

Comment:
How about either 'access to your public posts, their interactions and related data' or 'access to your public posts, including interactions and related data'?